src/components/PopupWithForm.js

```diff
@@ -7,6 +7,7 @@ export default class PopupWithForm extends Popup {
     this._inputList = Array.from(this._popupForm.querySelectorAll("input"));
     this._handleFormSubmit = handleFormSubmit;
     this._validator = validator;
+    this._submitButton = this._popupForm.querySelector(".model__submit-button");
   }
 
   close() {
```

Comment:
You shouldn’t create the **same methods** of the parent class if you don’t change them inside and only call them with `super`. The parent methods are available in any inheritor of the class.